packages/core/src/App/Containers/AccountSignupModal/account-signup-modal.jsx

```diff
@@ -62,6 +62,7 @@ const AccountSignup = ({ enableApp, isModalVisible, clients_country, onSignup, r
             setApiError(error);
         } else {
             isModalVisible(false);
+            sessionStorage.removeItem('signup_query_param');
```

Comment:
@suisin-deriv do we need to use SessionStore.set, SessionStore.remove etc.?
can we directly use sessionStorage in our project?

Reply:
I can see in the code base there's SessionStore which is there but not used else where. sessionStorage was mostly use around the code base

Reply:
Let me try to use sessionStorage 👌